Mathlib/MeasureTheory/Measure/MeasureSpaceDef.lean

```diff
@@ -94,6 +94,11 @@ instance Measure.instFunLike [MeasurableSpace α] : FunLike (Measure α) (Set α
 
 #noalign measure_theory.measure.has_coe_to_fun
 
+instance Measure.instOuterMeasureClass [MeasurableSpace α] : OuterMeasureClass (Measure α) α where
```

Comment:
I'm wondering if you really want to declare this instance, or have a `MeasureClass` (that you would also instantiate for `FiniteMeasure`) and an instance that a `MeasureClass` is also an `OuterMeasureClass`?

Reply:
`FiniteMeasure`s take values in `NNReal`s, not `ENNReal`s, see https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Measure/FiniteMeasure.html#MeasureTheory.FiniteMeasure.instCoeFun
AFAICT, `Measure` would be the only instance for `MeasureClass`, so I don't think that we should add this class.

Reply:
Good point, I agree!